skipper.env

```diff
@@ -128,4 +128,12 @@ INGRESS_VIP
 WORKERS_COUNT
 MASTERS_COUNT
 IPXE_BOOT
-SUBSYSTEM_LOCAL_REGISTRY
\ No newline at end of file
+SUBSYSTEM_LOCAL_REGISTRY
+NUTANIX_USERNAME
+NUTANIX_PASSWORD
+NUTANIX_ENDPOINT
+NUTANIX_PORT
+NUTANIX_CLUSTER
+NUTANIX_SUBNET
```

Comment:
```suggestion
NUTANIX_SUBNET_UUID
```

Reply:
This is not subnet uuid, this si the subnet name

Reply:
Right, lets rename it to SUBNET_NAME, as installer uses UUID for instance